go.mod

```diff
@@ -0,0 +1,71 @@
+module github.com/giantswarm/aws-operator
+
+go 1.13
+
+require (
+	github.com/aws/amazon-vpc-cni-k8s v1.6.0
+	github.com/aws/aws-sdk-go v1.29.32
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
+	github.com/giantswarm/apiextensions v0.2.0
+	github.com/giantswarm/apprclient v0.2.0
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/certs v0.2.0
+	github.com/giantswarm/e2e-harness v0.2.0
+	github.com/giantswarm/e2eclients v0.2.0
+	github.com/giantswarm/e2esetup v0.1.0
+	github.com/giantswarm/e2etemplates v0.2.0
+	github.com/giantswarm/e2etests v0.1.0
+	github.com/giantswarm/errors v0.2.0
+	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/helmclient v0.2.0
+	github.com/giantswarm/ipam v0.2.0
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/k8scloudconfig v0.0.0-20200325205340-f571257e51f7
```

Comment:
Well, this is complete bullshit. You can't define `v6.0.0`. Don't know right now how the go folks thought this should be.

Reply:
Sorted.